src/functions/makeAirtableBackup.js

```diff
@@ -3,7 +3,7 @@ import { fetchDataFromAirtable } from '../utils/airtableParser';
 import { uploadBackup } from '../utils/backup';
 
 export const handler = async (event, context, callback) => {
-  if (!event.hasOwnProperty('AIRTABLE_BASE') || !event.hasOwnProperty('AIRTABLE_TABLES') || !event.hasOwnProperty('S3_DIRECTORY')) {
+  if (!Object.prototype.hasOwnProperty.call(event, 'AIRTABLE_BASE') || !Object.hasOwnProperty.call(event, 'AIRTABLE_TABLES') || !Object.hasOwnProperty.call(event, 'S3_DIRECTORY')) {
```

Comment:
Could an example be made in the doc (README) on how to use this?

Reply:
This change has no functional impact; I made it because the old code failed the [no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins) rule.